### PR TITLE
sdxl h100 trt truss

### PR DESCRIPTION
**Pull request by pankajroark**
todo: Use templates to avoid duplication across various sdxl trt trusses. Let me do that in a follow-up.

### CODE REVIEW

**Review by aspctu — APPROVED** [review]
We should figure out the right story with templates in truss-examples and the templates inside of truss. Can be done off-line / follow-up.

File: stable-diffusion/stable-diffusion-xl-1.0-trt-h100/config.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    allow_patterns:
+      - "*.json"
+      - "*.fp16.safetensors"
+      - sd_xl_base_1.0.safetensors
+  - repo_id: stabilityai/stable-diffusion-xl-refiner-1.0
+    allow_patterns:
+      - "*.json"
+      - "*.fp16.safetensors"
+      - sd_xl_refiner_1.0.safetensors
+python_version: py39
```
Comment (joostinyi) [inline stable-diffusion/stable-diffusion-xl-1.0-trt-h100/config.yaml:34]: let's remove this – we use base image feature anyway

File: stable-diffusion/stable-diffusion-xl-1.0-trt-h100/config.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+system_packages:
+- python3.10-venv
```
Comment (joostinyi) [inline stable-diffusion/stable-diffusion-xl-1.0-trt-h100/config.yaml:36]: maybe this should be 3.11 too

**Review by joostinyi — APPROVED** [review]

File: stable-diffusion/stable-diffusion-xl-1.0-trt-h100/config.yaml (hunk 2 above)
Comment (pankajroark) [reply stable-diffusion/stable-diffusion-xl-1.0-trt-h100/config.yaml:36]: tensorrt seems to want 3.10

**Pull request merged**